tests/output_parsers/test_selection.py

```diff
@@ -26,3 +27,19 @@ def test_parse(output_parser: SelectionOutputParser) -> None:
     assert len(parsed.parsed_output) == 2
     assert parsed.parsed_output[0].choice == 1
     assert parsed.parsed_output[0].reason == "just because"
+
+
+def test_bugfix_llm_output_with_string_outside_of_json_issue_3135(
```

Comment:
let's not name this after an issue, and more just test_selection_parse_to_dict or something equivalent